build.sbt

```diff
@@ -174,6 +175,7 @@ lazy val `material-ui` =
       webpackDevServerPort := 8008,
       stFlavour := Flavour.Slinky,
       stReactEnableTreeShaking := Selection.All,
+      stIgnore += "react-proxy",
```

Comment:
I know I havent done this, but if you add a comment why it's ignored that would be nice.

Reply:
The hot reloading mechanism requires the `react-proxy`, which doesn't have the typescript definitions, without this, the compilation fails, in any case, I'll add a comment.